### PR TITLE
Add 8.3.6, 8.3.5, 8.2.18 and 8.1.28

### PR DESCRIPTION
**Pull request by mzch**
Add 8.3.5, 8.2.18 and 8.1.28

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -74,8 +74,8 @@ jobs:
           # PHP 7.3
           - { distro: 'ubuntu:22.04', php: '7.3.33'}
           # PHP 8.3
-          - { distro: 'centos:7', php: '8.3.4'}
-          - { distro: 'rockylinux:8', php: '8.3.4'}
+          - { distro: 'centos:7', php: '8.3.5'}
+          - { distro: 'rockylinux:8', php: '8.3.5'}
```
Comment (GrahamCampbell) [inline .github/workflows/ci.yml:78]: Incorrect vresion here.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (mzch) [reply .github/workflows/ci.yml]: fixed. thanks!

Comment (mzch) [reply .github/workflows/ci.yml]: fixed. thanks!

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]
Please remove the `.ci.yml.swp` file, or make the fork editible so I can do it.

**Pull request merged**